src/main/kotlin/com/cultureamp/eventsourcing/RelationalDatabaseEventStore.kt

```diff
@@ -126,10 +101,7 @@ class RelationalDatabaseEventStore<M : EventMetadata> @PublishedApi internal con
                             SequencedEvent(event, -1)
                         } else {
                             val insertedSequence = insertResult[eventsSinkTable.sequence]
-                            eventsSequenceStats.upsert {
-                                it[eventsSequenceStats.eventType] = eventType
-                                it[eventsSequenceStats.sequence] = insertedSequence
-                            }
+                            eventsSequenceStats?.update(event.domainEvent::class, insertedSequence)
```

Comment:
This is for API consistency on EventsSequenceStats - it has both the `update` and `lastSequence` methods - the latter also takes classes rather than strings, and uses the eventTypeResolve to transform from strings to classes